force-app/main/default/classes/ProgramController.cls

```diff
@@ -1,6 +1,13 @@
 public with sharing class ProgramController {
     @TestVisible
     private static ProgramService service = new ProgramService();
+    @TestVisible
+    private static FieldSetService fieldSetService = new FieldSetService();
+    private static final String ENGAGEMENT_FIELD_SET = 'CreateProgramEngagement';
+    private static final String CONTACT_FIELD_SET = 'CreateContact';
+    private static final String PROGRAM_COHORTS_KEY = 'programCohorts';
```

Comment:
I think maybe the PROGRAM_COHORTS_KEY is no longer needed with the (totally rad) updates?